packages/components/scrollbar/src/scrollbar.vue

```diff
@@ -1,6 +1,12 @@
 <template>
   <div ref="scrollbarRef" :class="ns.b()">
-    <div ref="wrapRef" :class="wrapKls" :style="style" @scroll="handleScroll">
+    <div
+      ref="wrapRef"
+      :class="wrapKls"
+      :style="style"
+      :tabindex="props.tabindex"
```

Comment:
```suggestion
      :tabindex="tabindex"
```

Writing like this looks redundant

Reply:
Fixed